tests/zfs-tests/tests/functional/cli_root/zfs_receive/zfs_receive_013_pos.ksh

```diff
@@ -0,0 +1,83 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2016, [name of copyright owner]. All rights reserved.
```

Comment:
You wrote the test case, feel free to add your name here.  Or your companies name if that's more appropriate.